### ISSUE TITLE
Remove X, Y, and Z constants from Sketch and Py5Graphics

### ISSUE THREAD

**Issue by hx2A** [py5coding/py5generator#105]
They aren't useful and they trip up Thonny users or anyone using imported mode. Nobody can use them as variable names.

**Comment by hx2A** [comment]
There more constants that I believe are only used internally to Processing. These all deserve a closer look to see if they should really be included:

OTHER
WINDOWS
MACOS
LINUX
TIFF
TARGA
JPEG
GIF
DEFAULT_WIDTH
DEFAULT_HEIGHT
EXTERNAL_MOVE
EXTERNAL_STOP
HINT_COUNT
CHATTER
COMPLAINT
PROBLEM

**Comment by hx2A** [comment]
A few more in Py5Graphics only:

ALPHA_MASK
RED_MASK
GREEN_MASK
BLUE_MASK

**Comment by hx2A** [comment]
So what's going on here is when I first created the [data files](https://github.com/py5coding/py5generator/tree/main/py5_resources/data) that py5 uses to determine which of Processing's public variables and methods should be made available in py5, I had to make a lot of decisions quickly about what should be included. While writing the reference documentation I had the opportunity to review all of those decisions about the methods only. The constants don't have documentation so they did not get the same careful review. I went through all of them now and made the necessary fixes to remove constants that should not be there.
